### DISCUSSION TITLE
Unique URI in all defined routes

### DISCUSSION THREAD

**Discussion by lostika86** [statamic/cms#4837] [Q&A]
Does Statamic validates URI uniqueness against routes, which are defined in `routes/web.php`?

Goal: disable entry creation with slug `blog`.

Logic: 
`blog` slug is already defined in `routes/web.php` as `Route::statamic('blog','blog_items_listing.antlers.html',['layout' => 'layout_custom_blog'])`;
If user creates an entry with `blog` slug, and `UniqueEntryValidation` is defined, statamic saves it with slug `blog`.

Is there any solution how to disallow save URIs which are defined globally?

**Comment by duncanmcclean** [comment]
Make a blank page entry where the URL is `/blog` and set the custom layout/template on that: 

```yaml
layout: layout_custom_blog
template: blog_items_listing
```

The `UniqueEntryValidation` rule only cares about the URLs of other entries, it doesn't take custom routes into consideration.

**Reply by lostika86** [reply]
Yes, it can be a solution. Honestly, I do not want to create entry for this. 
example: I have collection for blog entries, which I plan to relate to '/blog', but I want to eliminate deleting '/blog' - this is the reason why I do not want to create entry for `/blog`
